README.md

```diff
@@ -1,137 +1,167 @@
+
+#Stryker Karma Runner
+
+A plugin to use the karma test runner in [Stryker](https://stryker-mutator.github.io), the JavaScript mutation testing framework
+
+## Warning
+
+The stryker-karma-runner is available from stryker v0.4.0 onward.
+
+## Install
+
+Install stryker-karma-runner from your project folder:
+
+```bash
+npm i --save-dev stryker-karma-runner
+```
+
+## Peer dependencies
+
+The `stryker-karma-runner` is a pluggin for `stryker` to enable `karma` as a test runner. As such you should install the correct versions of the dependencies:
+
+* `karma`
+* `stryker-api`
+
+For the current versions, see the `peerDependencies` section in the [package.json](https://github.com/stryker-mutator/stryker-karma-runner/blob/master/package.json).
+
+These are marked as `peerDependencies` of `stryker-karma-runner` so you get a warning during installation when the correct versions are not installed.
+*Note*: Karma itself also requires some plugins to work.  
+
+## Configuring
+
+You can either configure the karma test runner from the `stryker.conf.js` file or from the command line. This readme describes how to do it via the config file.
+
+### Load the plugin
+
+In order to use the `stryker-karma-runner` it must me loaded in the stryker mutation testing framework via the stryker configuration. 
+Easiest is to *leave out* the `plugins` section from your config entirely. That way, all node_modules starting with `stryker-` will be loaded.
+
+If you do decide to choose specific modules, don't forget to add `'stryker-karma-runner'` to the list of plugins to load.
+
+### Use the test runner
+
+Specify the use of the karma testRunner: `testRunner: 'karma'`.
+
+### Karma config
+
+#### Automatic setup
+
+You can configure stryker to use *your* `karma.conf.js` file. 
+
+```javascript
+// Stryker.conf.js
+module.exports = function (config) {
+    config.set({
+        testRunner: 'karma',
+        testFramework: 'jasmine', // <-- add your testFramework here
+        karmaConfigFile: 'karma.conf.js' // <-- add your karma.conf.js file here
+        mutate: [
+            'src/**/*.js' // <-- mark files for mutation here
+        ]
+    });
+}
+```
+
+This will configure 3 things for you:
+
+* Karma's `files` option will be used to configure the files in Stryker, you don't need to keep a list of files in sync in both `stryker.conf.js` and `karma.conf.js`.
+* Karma's `exclude` option will be used to ignore files in Stryker (using `!` to ignore them)
+* Other karma config will be copied to the `karmaConfig` option in stryker config. These will be used by the `stryker-karma-runner` during mutation testing.
+    * **Note**: Any manual setup you configure in the `karmaConfig` options will *not* be overwritten.
+
+#### Manual setup
+
+**Note**: Using the manual setup will not read your `karma.conf.js` options. You'll probably need to *Override karma config* (see next section)

```

Comment:
**Note**: Using the manual setup will not read your `karma.conf.js`. You'll probably need to *override the karma config* (see next section).